hdl/modules/wb_position_calc/cheby/wb_pos_calc_regs.cheby

```diff
@@ -2199,3 +2199,150 @@ memory-map:
             access_bus: READ_WRITE
             access_dev: READ_ONLY
             clock: fs_clk2x_i
+# ---------------------------------------------
+# -----------  ADC Offset Registers -----------
+# ---------------------------------------------
+  - reg:
+      name: adc_ch0_swclk_0_offset
+      address: 0x00000140
+      width: 32
+      access: rw
+      description: ADC channel 0 offset on RFFE switch state 0 (inverted)
+      comment: |
+        ADC channel 0 offset on RFFE switch state 0 (inverted)
```

Comment:
Should be mentioned here that these offset registers are in two's complement and are subtracted from the ADC sample.